include/nbl/video/alloc/SubAllocatedDescriptorSet.h

```diff
@@ -248,22 +258,34 @@ class SubAllocatedDescriptorSet : public core::IReferenceCounted
 		{
 			eventHandler.wait(maxWaitPoint, unallocatedSize);
 
-			unallocatedSize = try_multi_allocate(binding, unallocatedSize, &descriptors[count - unallocatedSize], &outAddresses[count - unallocatedSize]);
+			unallocatedSize = try_multi_allocate(
+				binding, 
+				unallocatedSize, 
+				&descriptors[count - unallocatedSize], 
+				&outDescriptorWrites[count - unallocatedSize], 
+				&outAddresses[count - unallocatedSize]
```

Comment:
another reason not to take a list of descriptors, writes and addresses to allocate

gaps might be in weird places, you're  sure to write them out wrong

Reply:
done in https://github.com/Devsh-Graphics-Programming/Nabla/pull/662